src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -108,6 +130,12 @@ export const Chatbox: React.FC<ChatboxProps> = ({
             placeholder="Write your message..."
             autoComplete="off"
           />
+          <span
+            className="Chatbox__emoji-picker-toggler"
+            onClick={toggleEmojiPicker}
+          >
+            😃
+          </span>
```

Comment:
Would this be better as a button (with `type="button"`) given we're doing `onClick` things with it?

Reply:
Was done in previous commits